apps/design-system-docs/wrangler.jsonc

```diff
@@ -0,0 +1,8 @@
+{
+	"name": "preview",
+	"compatibility_date": "2025-05-28",
```

Comment:
We might need to update some docs with regards to how/when this should be updated: 

https://developers.cloudflare.com/workers/configuration/compatibility-dates/#setting-compatibility-date